bin/iuf-skopeo-sync

```diff
@@ -59,22 +61,27 @@ process(){
         return 1
     fi
 
-    # Remove the leading path element. 
-    ns_path=$(echo $docker_dir|cut -d'/' -f2-)
+    # Ensure $docker_dir ends with exactly one slash
+    docker_dir="${docker_dir%%+(/)}/"
 
     if skopeo sync --scoped \
         --src dir --dest docker \
         --dest-creds "${NEXUS_USERNAME:-admin}:${NEXUS_PASSWORD}" \
         --dest-tls-verify=false \
-        /images/"$docker_dir" ${NEXUS_REGISTRY:-registry.local}/$ns_path; then
+        /images/"$docker_dir" "${NEXUS_REGISTRY:-registry.local}"; then
 
         # Record results for each image. Note that we have to search the directory for
         # things that appear to be docker images here, which unfortunately is not directly
         # looking at the result of the `skopeo sync` command.
         find "/images/$docker_dir" -name "manifest.json" | while read -r manifest_file; do
-            image_name_and_version=$(basename "$(dirname "$manifest_file")")
-            image_name="$(cut -d":" -f 1 <<< "$image_name_and_version")"
-            image_version="$(cut -d":" -f 2 <<< "$image_name_and_version")"
+            # Get the path to the image relative to the specified $docker_dir, as that
+            # is the name of the image that was uploaded. For example, if images are found
+            # in /images/docker/cray/IMAGE:VERSION and $docker_dir is 'docker/', images were
+            # uploaded as "cray/IMAGE:VERSION".
+            relative_image_dir="${manifest_file/\/images\/${docker_dir}/}"
```

Comment:
seems like it might be very slightly more clear and achieve the same goal to use something like this:

```suggestion
            relative_image_dir="${manifest_file##/images/${docker_dir}}"
```